docs/user-guide/migrating-from-jscs.md

```diff
@@ -0,0 +1,79 @@
+
+
+
+## Convert Configuration Files Using Polyjuice
+
+[Polyjuice](https://github.com/brenolf/polyjuice) is a utility for converting JSCS (and JSHint) configuration files into ESLint configuration files automatically. It understands the equivalent rules from each utility and will automatically output an ESLint configuration file that is a good approximation of your existing JSCS file.
+
+To install Polyjuice:
+
+```
+$ npm install -g polyjuice
+```
+
+To convert your configuration file, pass in the location of your `.jscs` file using the `--jscs` flag:
+
+```
+$ polyjuice --jscs .jscsrc > .eslintrc
+```
+
+This creates a `.eslintrc` with the equivalent rules from `.jscsrc`.
+
+If you have multiple `.jscsrc` files, you can pass them all and Polyjuice will combine them into one `.eslintrc` file:
+
+```
+$ polyjuice --jscs .jscsrc ./foo/.jscsrc> .eslintrc
+```
+
+**Note:** If you'd like to create a new configuration file from scratch, you can run `eslint --init`, which will guide you through the process of creating a new configuration file.
+
+## Converting Presets
+
+There are shareable configs available for most JSCS presets. The equivalent shareable configs for each JSCS preset are listed in the following table:
+
+| **JSCS Preset** | **ESLint Shareable Config** |
+|-----------------|-----------------------------|
+| `airbnb`        | [`eslint-config-airbnb`](https://npmjs.com/package/eslint-config-airbnb) |
+| `crockford`        | (not available) |
+| `google`        | [`eslint-config-google`](https://npmjs.com/package/eslint-onfig-google) |
+| `grunt`        | [`eslint-config-grunt`](https://npmjs.com/package/eslint-config-grunt) |
+| `idiomatic`        | [`eslint-config-idiomatic`](https://npmjs.com/package/eslint-config-idiomatic) |
+| `jquery`        | [`eslint-config-jquery`](https://npmjs.com/package/eslint-config-jquery) |
+| `mdcs`        | (not available) |
+| `node-style-guide`        | [`eslint-config-node-style-guide`](https://npmjs.com/package/eslint-config-node-style-guide) |
+| `wikimedia`        | [`eslint-config-wikimedia`](https://npmjs.com/package/eslint-config-wikimedia) |
+| `wordpress`        | [`eslint-config-wordpress`](https://npmjs.com/package/eslint-config-wordpress) |
+
+As an example, suppose that you are using the `airbnb` preset, so your `.jscsrc` file looks like this:
+
+```json
+{
+    "preset": "airbnb"
+}
+```
+
+In order to get the same functionality in ESLint, you would first need to install the `eslint-config-airbnb` shareable config package:
+
+```
+$ npm install eslint-config-airbnb --save-dev
```

Comment:
That wouldn't work like that though

I tried it with
```sh
npm i eslint-config-airbnb eslint-plugin-import@latest eslint-plugin-jsx-a11y@latest --save-dev
```

as error output while installing `eslint-config-airbnb` suggested, but that didn't work either :/.

This kinda works -
```sh
npm install eslint-config-airbnb-base eslint-plugin-import@latest --save-dev
```

With `"extends": "airbnb"` -> "extends": "airbnb-base". Although i'm not sure if this is fare replacement.
/cc @ljharb 

Also you might want to mention if plugin is installed locally, but you execute eslint globally that would also don't work. That is a big difference with jscs, since it used locally installed plugins/configs even if installed only globally.

All this could be solved with https://github.com/eslint/eslint/issues/3458 though